charts/refinery/templates/deployment-redis.yaml

```diff
@@ -16,12 +19,23 @@ spec:
   {{- include "refinery.redis.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
+      {{- if or .Values.podAnnotations .Values.redis.podAnnotations }}
       annotations:
+      {{- with .Values.podAnnotations }}
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
I am not entirely sure why we would want to use `.Values.podAnnotations` for redis pods but keeping this for backward compatibility.

Reply:
Can you add a comment above the condition specifying this backwards compatibility as the reason why both are used?

Reply:
Done